package/Dockerfile.shipyard-system-base

```diff
@@ -0,0 +1,34 @@
+FROM fedora:32
+
+ENV DAPPER_HOST_ARCH=amd64 SHIPYARD_DIR=/opt/shipyard
+ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH}
+
+# Requirements:
+# Component      | Usage
+# -----------------------------------------------------------
+# gcc            | needed by `go test -race` (https://github.com/golang/go/issues/27089)
+# git            | find the workspace root
+# curl           | download other tools
+# moby-engine    | Dapper (Docker)
+# golang         | build
+# kubectl        | e2e tests (in kubernetes-client)
+# golangci-lint  | code linting
+# helm           | e2e tests
+# kind           | e2e tests
+# ginkgo         | tests
+# goimports      | code formatting
+# make           | OLM installation
+# findutils      | validate (find go packages)
+# subctl *       | Submariner's deploy tool (operator)
+# upx            | binary compression
+# jq             | JSON processing (GitHub API)
+# diffutils      | required for goimports
+# ShellCheck     | shell script linting
+RUN dnf -y install --nodocs --setopt=install_weak_deps=False \
```

Comment:
Why don't we keep this part as a more static first layer of the image, and then... a 2nd run with all the less-static parts?

I prefer the single image approach if that works, limiting complexity to some extent.

Reply:
I second that approach.

Reply:
> Why don't we keep this part as a more static first layer of the image, and then... a 2nd run with all the less-static parts?

Because I tried that and it doesn’t work, with this image at least. The intermediary layers are discarded and never re-used when building updated images.

Reply:
I've investigated it and proposed a solution at #244 :)

Reply:
You’re a star @mkolesnik, I’ll check that out and see how it behaves.